src/d3a/models/state.py

```diff
@@ -130,17 +130,16 @@ def free_storage(self, time_slot):
         """
         in_use = self._used_storage \
             - self.pledged_sell_kWh[time_slot] \
-            + self.pledged_buy_kWh[time_slot] \
```

Comment:
Ok, so to summarise, free_storage property should not include the amount of offered energy for buy/sell , right ? I am of course concerned that this might affect many things, but let's give it a try that way.

Reply:
The reason I did it this way is `free_storage` & `used_storage` doesn't have the same pattern of accounting energy, meaning `free_storage` takes into account the `promised energy that hasn't been traded` whereas `used_storage` is not doing so

Reply:
I see, then I agree that these 2 properties should complement each other. Thanks for finding this!